### PR TITLE
ft-navbar

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Added menu to navbar at 580px breakpoint (main add)
- Though implementation of the navbar is not complete as we await the finished UI prototype

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Do a git fetch and checkout into this local branch. Do an `npm install` and run the pr

#### What are the relevant tickets?
- [WEB-102](https://airqoteam.atlassian.net/browse/WEB-102)

#### Screenshots (optional)
menu-navbar

### CODE REVIEW

**Comment by Codebmk** [comment]
snippet-1
snippet-2

@ja-odur

File: website/frontend/static/frontend/main.bundle.js.LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
```
[the remark this answers is not in the document]
Comment (ja-odur) [reply website/frontend/static/frontend/main.bundle.js.LICENSE.txt:1]: Same applies for the `main.bundle.js` file.

**Review by ja-odur — COMMENTED** [review]
@Codebmk I have noticed the navbar bottom border (on figma) is not inplemented on this PR. You may want to add that

image

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by FaithDaka — APPROVED** [review]

**Pull request merged**